images/run-playbook/README.md

```diff
@@ -0,0 +1,43 @@
+# Containerized openshift-ansible to run generic playbooks
+
+This image uses the [playbook2image](https://github.com/aweiteka/playbook2image) source-to-image image to containerize `openshift-ansible`. The resulting image can run any of the provided playbooks.
+
+## Build
+
+1. **Docker**:
```

Comment:
the configuration for the container seems easy enough that should not be a problem to make a system container of it we will need to provide the configuration for runC.

We'll probably benefit of running the systemd service as Type=oneshot